### PR TITLE
Adding redirects to the yaml file for SEO.

### PR DESCRIPTION
**Pull request by SgtCoin**
I added redirects to the yaml file, using the same format as the current redirects. This redirect list comes from an external SEO team, but you can see the sites listed here that need redirects. 

[Redirect List](https://docs.google.com/spreadsheets/d/1xnZwXFPqnT0lQJz6rp1iaFuM2y86LHj46oIHSQIChec/edit?usp=sharing
)

I would like to get another person to review this to make sure its accurate.

### CODE REVIEW

File: .gitbook.yaml [hunk 1]
```diff
@@ -199,3 +199,201 @@ redirects:
...
+  store/lotus/retrieve-data: basics/how-retrieval-works.md
+  store/lotus/store-data: basics/what-is-filecoin/storage-model.md
+  store/lotus/very-large-files: basics/how-storage-works/filecoin-and-ipfs.md
+  store/overview: basics/what-is-filecoin/storage-model.md
+  store/slate: basics/what-is-filecoin.md
+  store/starling: basics/what-is-filecoin.md
+  store/tools/estuary: reference/general.md
+  store/tools/starling: basics/project-and-community/filecoin-faqs.md
+  uild/examples/sample-architectures: storage-providers/infrastructure/reference-architectures.md
+  uild/lotus/api-token-generation: reference/built-in-actors/protocol-api.md
```
Comment (johnnymatthews) [inline .gitbook.yaml:399]: Just wanna confirm that these are meant to start with `uild/...` and aren't typos. Are these two lines correct @SgtCoin ?

Comment (SgtCoin) [reply .gitbook.yaml]: Fixed it. It was a typo, good catch.

Comment (johnnymatthews) [reply .gitbook.yaml]: Danke!

**Comment by johnnymatthews** [comment]
Looks like GitBook is throwing some errors at us. Let me investigate.

image

**Pull request merged**